media-sound/cantata/cantata-3.3.0.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="Featureful and configurable Qt client for the music player daemon (MPD)"
+HOMEPAGE="https://github.com/nullobsi/cantata"
+SRC_URI="https://github.com/nullobsi/cantata/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="cdda cddb http-server mtp musicbrainz replaygain streaming +taglib udisks zeroconf"
+REQUIRED_USE="
+	cdda? ( taglib udisks || ( cddb musicbrainz ) )
+	cddb? ( cdda taglib )
+	mtp? ( taglib udisks )
+	musicbrainz? ( cdda taglib )
+	replaygain? ( taglib )
+	udisks? ( taglib )
+"
+# cantata has no tests
+RESTRICT="test"
```

Comment:
I'm not sure where that has its origins but it makes no sense to restrict no tests.